charts/kube-master/charts/etcd/values.yaml

```diff
@@ -19,4 +19,20 @@ resources:
   limits:
     cpu: 500m
     memory: 512Mi
-
+backup:
+  image:
+    repository: sapcc/etcdbrctl
+    tag: 0.3.0
+    pullPolicy: IfNotPresent
+  config:
+    schedule: "*/5 * * * *"
+    maxBackups: 10
```

Comment:
Please add some comment inline what these values mean

Reply:
@databus23 I added comments and changed default values. Backups are kept for one week now.